manifests/project.pp

```diff
@@ -56,14 +59,15 @@
   $custom_inst = true,
   $lock        = false,
   $user        = 'root',
+  $php_bin     = '',
 ) {
   include composer
 
   validate_re($ensure, '^(present|installed|latest)$', '$ensure must be one of present or latest.')
   validate_re($prefer, '^(dist|source)$', '$prefer can only be one of source or dist. See `composer install --help`.')
   validate_bool($dev, $scripts, $custom_inst)
 
-  $composer  = "${composer::target_dir}/${composer::command_name}"
+  $composer  = strip("${php_bin} ${composer::target_dir}/${composer::command_name}")
```

Comment:
what does strip do here?

Reply:
It removes the leading space if `php_bin` is unset.